account_reconcile_oca/static/src/js/reconcile_controller.esm.js

```diff
@@ -0,0 +1,99 @@
+/** @odoo-module */
+const {useState, useSubEnv} = owl;
+import {KanbanController} from "@web/views/kanban/kanban_controller";
+import {View} from "@web/views/view";
+import {useService} from "@web/core/utils/hooks";
+
+export class ReconcileController extends KanbanController {
+    async setup() {
+        super.setup();
+        this.state = useState({
+            selectedRecordId: null,
+        });
+        useSubEnv({
+            parentController: this,
+            exposeController: this.exposeController.bind(this),
+        });
```

Comment:
```suggestion
        });
        this.env.exposeController(this);
```
You should call "this.env.exposeController(this);" because when you enter in the reconciliation menu the default view is the kanban view that uses this controller and not the form one, so if you don't call it you will get an error that this.form_controller is undefined.

Reply:
It is called on the ReconcileFormController :thinking:

Reply:
Yep, but when you enter the view to reconcile, we are in kanban and does not jump only the function in ReconcileFormController, I tried it on my machine locally, if you want I'll try it again and pass you the video.